src/terminal.rs

```diff
@@ -436,22 +443,36 @@ impl Terminal {
 
                     let mut attrs = self.default_attrs;
 
+                    let cell_fg = if indexed.cell.flags.contains(Flags::BOLD) {
+                        as_bright(indexed.cell.fg)
```

Comment:
I wouldn't do this by default, since there is already an ANSI escape code to use the bright colors, and bold already sets a different font.